qutebrowser/browser/webkit/webview.py

```diff
@@ -104,6 +104,12 @@ def _set_bg_color(self):
         palette = self.palette()
         if col is None:
             col = self.style().standardPalette().color(QPalette.Base)
+        if self.view() is not None :
+            if col.alpha() < 255 :
+                print(col.alpha())
```

Comment:
This shouldn't be here

Reply:
I don't understand why it shouldn't be here ?

Reply:
Because you don't want qutebrowser to print out random numbers?

Reply:
Yes you're right. It's now modified. I read the code too fast.